tests/unit/test_charm.py

```diff
@@ -17,7 +17,7 @@ def harness():
     return Harness(Operator)
 
 
-@patch("charm.KubernetesServicePatch", lambda x, y: None)
+@patch("charm.KubernetesServicePatch", lambda *_, **__: None)
```

Comment:
I don't really understand this change :D

Reply:
Right, so the newwer version of the kubernetes_service_patch library made a change in `KubernetesServicePatch`, which now receives several different arguments. To mock the `KubernetesServicePatch`, the [docs suggest](https://github.com/canonical/observability-libs/blob/main/lib/charms/observability_libs/v1/kubernetes_service_patch.py#L119) to add a lambda with variables representing each of the arguments you used in the instantiation. With *_ and **__ we just pass ALL of them.

Reply:
Got it, thank you!